src/script_chainer/win_exe/script_runner.py

```
@@ -390,33 +431,62 @@ def run_script(
         return
 
     script_path = script_config.script_path
+    no_log_timeout = script_config.no_log_timeout_seconds
+    max_retries = script_config.no_log_max_retries if no_log_timeout > 0 else 0
+
+    # last_log_time[0] 记录最近一次收到子进程输出的时间（仅外部 EXE 脚本有效）
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)